### ISSUE TITLE
Missing or wrong/unnecessary icon in 0.17.11 guided tour

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/kestra#4284]
### Describe the issue

latest 0.17.11

![image](https://github.com/kestra-io/kestra/assets/86264395/08afb047-c460-4599-82e4-403d82785aab)

the same looks perfectly fine on develop:

![image](https://github.com/kestra-io/kestra/assets/86264395/cef6f0ff-2a92-4ae6-8e94-bb0e8066b77f)

@MilosPaunovic if you could briefly sanity-check this. thanks! 🙏 

### Environment

- Kestra Version:
- Operating System (OS/Docker/Kubernetes):
- Java Version (if you don't run kestra in Docker):

**Comment by MilosPaunovic** [comment]
I've added a way of filtering task icons - but, a bit of a nuisance here is that it does not happen on `develop` image, only on `0.17.x` (tested on `0.17.15`), so, we'll have to wait for releasing another version to test it in that manner. I've also added couple of precise `data` attributes which will help pinpoint the problem if this solution is not alright.

**TLDR**: Keeping this issue opened until next Docker version is deployed to test it.

**Comment by anna-geller** [comment]
perfect! I can confirm it works now on develop, so we may also close already as it should be then fixed with the release next week

![image](https://github.com/user-attachments/assets/ec2a9f49-ea45-4815-bb1d-8f2eafbe7cd2)

if you prefer to keep it open, please reopen 👍
